src/clr-angular/forms/select/option.spec.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {Component, ElementRef} from "@angular/core";
+
+import {TestContext} from "../../data/datagrid/helpers.spec";
+import {POPOVER_HOST_ANCHOR} from "../../popover/common/popover-host-anchor.token";
+import {IfOpenService} from "../../utils/conditional/if-open.service";
+import {ClrOption} from "./option";
+
+@Component({
+    template: `
+        <clr-option>
+            Test
+        </clr-option>
+    `,
+    providers: [IfOpenService, {provide: POPOVER_HOST_ANCHOR, useExisting: ElementRef}]
+})
+class TestComponent {
+}
+
+export default function(): void {
+    describe("Select Option Component", function() {
+        let context: TestContext<ClrOption, TestComponent>;
+        let ifOpenService: IfOpenService;
+
+        beforeEach(function() {
+            context = this.create(ClrOption, TestComponent, [IfOpenService]);
```

Comment:
Do we need `IfOpenService` provided here again? I see it's provided at the component level above?

Reply:
Dont think we do. Will remove it. Thanks!